graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDeflectorResource.java

```diff
@@ -20,6 +20,6 @@
 import retrofit2.http.POST;
 
 public interface RemoteDeflectorResource {
-    @POST("/system/deflector/cycle")
+    @POST("system/deflector/cycle")
```

Comment:
We agreed to simply fix the affected resources in the relevant plugins.
Since the default is to run the API on a sub path, this will fail early on which is good enough.